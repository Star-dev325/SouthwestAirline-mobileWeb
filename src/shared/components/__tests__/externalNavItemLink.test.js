import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import React from 'react';
import ExternalNavItemLink from 'src/shared/components/externalNavItemLink';

describe('ExternalNavItemLink', () => {
  describe('render', () => {
    it('should have correct props', () => {
      const { container } = createComponent();

      expect(container).toMatchSnapshot();
    });
  });
});

function createComponent() {
  return render(
    <ExternalNavItemLink className={'className'} disabled href="test-href">
      <div>
        <span>Hi</span>
      </div>
    </ExternalNavItemLink>
  );
}
