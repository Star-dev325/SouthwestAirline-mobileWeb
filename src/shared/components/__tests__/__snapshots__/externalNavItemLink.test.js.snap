// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExternalNavItemLink render should have correct props 1`] = `
<div>
  <a
    class="nav-item-link className nav-item-link_disabled"
    href="test-href"
    target="_blank"
  >
    <div>
      <span>
        Hi
      </span>
    </div>
    <i
      class="nav-item-link--icon icon icon_keyboard-arrow-right"
    />
  </a>
</div>
`;
